README.md

```diff
@@ -146,6 +146,29 @@ especially in the case in which the exporter is external to the code it's monito
 
 For more detailed documentation on [Availability exporters](https://gitlab.cee.redhat.com/konflux/docs/documentation/-/blob/main/o11y/monitoring/availability_exporters.md?ref_type=heads)
 
+## Recording Rules
+
+Recording rules allow us to precompute frequently needed or computationally expensive expressions
+and save their result as a new set of time series. Recording rules are the go-to approach for
+speeding up the performance of queries that take too long to return. When other teams want to go
+with their own metrics format for exporters they need to adapt to desired metric form by
+translating it using a recording rule. 
+
+These recording rules should be put in the [rhobs/recording folder](https://github.com/redhat-appstudio/o11y/tree/main/rhobs). 
+
+The standard format is single availability metric `konflux_up` with labels `service` and `check`.
+Each time series will have the service and check labels for the name of the originating service
+and availability check it performed, respectively. The metric konflux_up should return either 0
+or 1 based on the availability of the component/service. If the service is up then the metric
+should return 1 else 0.
+
+Recording rule example provided here has below format
```

Comment:
We should include the link to the reference rule.

Reply:
Added the link for reference rule.